### DISCUSSION TITLE
如何在X6中 鼠标移入事件中 实现 tooltip 提示框的功能

### DISCUSSION THREAD

**Discussion by zzh422** [antvis/X6#2229] [Q&A]
如何在X6中 鼠标移入事件中 实现 tooltip 提示框的功能

**Comment by journey0726** [comment]
说一种简单点的，提前写好一个tooltip组件，将它的位置设置在屏幕之外，然后监听鼠标移入事件，将tooltip组件的坐标变成现在鼠标的坐标。
